servers/physics_2d/broad_phase_2d_hash_grid.h

```diff
@@ -51,6 +51,9 @@ class BroadPhase2DHashGrid : public BroadPhase2DSW {
 		CollisionObject2DSW *owner;
 		bool _static;
 		Rect2 aabb;
+		// Owner's collision_mask/layer, used to detect changes in layers
```

Comment:
Comments need a period at the end (coding style).

Reply:
You'd think I'd learn after the number of times I've gotten this comment